stories/rich-text-editor/rich-text-editor.stories.js

```diff
@@ -0,0 +1,1116 @@
+				<li id="overflowRTEFont-currentlyFocusedItem" class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Select</span>
+				</li>
+				<li aria-selected="true" class="fd-list__item is-selected" role="option" tabindex="0">
+				    <span class="fd-list__title">Andale Mona</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Arial</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Arial Black</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Book Antiqua</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Comic Sans MS</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Courier New</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Georgia</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Helvetica</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Impact</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Symbol</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Tahoma</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Terminal</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Times New Roman</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Trebuchet MS</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Verdana</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Webdings</span>
+				</li>
+				<li class="fd-list__item" role="option" tabindex="0">
+				    <span class="fd-list__title">Wingdings</span>
+				</li>
+			    </ul>
+			</div>
+		    </div>
+		    <div class="fd-popover">
+			<div class="fd-popover__control">
+			    <div class="fd-select fd-select--compact">
+				<button aria-expanded="false" aria-haspopup="listbox" aria-label="select-label" class="fd-select__control" id="overflowRTEFontSize" onclick="
+				    toggleElAttrs('h0C6A3217', ['aria-hidden']);
+				    toggleElAttrs('overflowRTEFontSize', ['aria-expanded']);
+				    " value="List Item 1" tabindex="0">
+				<span is="overflowSelectValue" class="fd-select__text-content">10 pt</span>
```

Comment:
`id` typo? Please check in other places too

Reply:
modified.